data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,542 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	"Oh, hello again <first>!" Anna says. "Is the ship ready?"`
+			`	Hearing an affirmative answer, Anna stands up from her chair and begins following you to the ship. She settles into her bunk as you prepare for a trip to the <destination>.`
+				accept
+	on enter "Quaru"
+		conversation
+			`As you enter <system>, Anna walks out of her bunk and finds a spot on the side of the bridge, peering into the empty void. "What a massive ringworld!" she exclaims.`
+			choice
+				`	"It's enormous. As you know, they stole it from the Quarg roughly seven thousand years ago."`
+				`	(Say nothing.)`
+			`	"This is going to be a great source of inspiration for my next novel," says Anna, still in awe.`
+	on complete
+		conversation
+			`Anna is amazed as you touch down on the ringworld. As the two of you open the hatch, you see a Heliarch agent in front of you.`
+			`	"Greetings, humans. For you, what can I do today?" asks the Heliarch. You respond that you would like to register Anna with the Heliarchy. After reaching the registration office, Anna enters it, coming out after fifteen minutes. "They finally got all the paperwork done. I'm off to the spaceport to get some food. Meet me there to discuss our next move."`
+
+
+mission "Coalition: Struggling Author 3"
+	name "Back to <planet>"
+	description "Now that Anna Fangvin has completed the registration process, help her get a translation device on <destination>."
+	source "Ring of Friendship"
+	destination "Ki Patek Ka"
+	to offer
+		has "Coalition: Struggling Author 2: done"
+	on offer
+		conversation
+			`Going through the spaceport looking for Anna, you finally find her at a restaurant serving food made for Saryds. She's typing away furiously at her laptop, with a plate of vegetables next to her. She sees you and motions for you to take a seat.`
+			`	You sit down next to Anna. "These vegetables are so delicious! After months of instant noodles and takeout back home, it finally feels good to eat something that's actually healthy." she exclaims, closing the computer.`
+			choice
+				`	"How good is it?"`
+					goto good
+				`	"What did you usually eat back in human space?"`
+			`	"My diet wasn't exactly the best. All I had was carbs and takeout food; I never could find time in my schedule to cook at home. I was working hard on 'Beyond the Hyperdrive' at the time. This food is better than anything I could ever find on Zug."`
+			choice
+				`	"What's our plan now?"`
+					goto published
+				`	"What are we going to do next?"`
+					goto published
+			label good
+			`	"It's very good. Probably better than anything I've had back on Zug. Now, I'd like to get my book published here."`
+				goto company
+			label published
+			`	"I want to get my book published here."`
+			label company
+			`	"Have you found a publishing company yet?" you ask. "No." Anna then begins complaining about how it was extremely difficult for her to communicate with the restaurant staff, who did not have a translation box.`
```

Comment:
```suggestion
			`	"Have you found a publishing company yet?" you ask.`
			`"No," Anna replies, then begins complaining about how it was extremely difficult for her to communicate with the restaurant staff, who did not have a translation box.`
```

Otherwise it makes it look like the player says no.